tests/src/test/java/alluxio/client/cli/fs/command/LsCommandIntegrationTest.java

```diff
@@ -290,6 +290,52 @@ public void lsWithSortByPath() throws IOException, AlluxioException {
         "            100   NOT_PERSISTED .+ .+ 100% /testRoot/testLongFile");
   }
 
+  /**
+   * Tests ls command with sort by access time option.
+   */
+  @Test
+  @LocalAlluxioClusterResource.Config(
+      confParams = {PropertyKey.Name.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false",
+          PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE, "NOSASL",
+          PropertyKey.Name.MASTER_FILE_ACCESS_TIME_UPDATE_PRECISION, "0"})
+  public void lsWithSortByAccessTime() throws IOException, AlluxioException {
+    String oldFileRecentlyAccessed = "/testRoot/testFileRecent";
+    String oldFileName = "/testRoot/testFile";
+    FileSystemTestUtils
+        .createByteFile(mFileSystem, oldFileRecentlyAccessed, WritePType.MUST_CACHE, 10);
+    FileSystemTestUtils
+        .createByteFile(mFileSystem, oldFileName, WritePType.MUST_CACHE, 10);
+
+    FileSystemTestUtils.loadFile(mFileSystem, oldFileRecentlyAccessed);
+    mFsShell.run("ls", "--sort", "lastAccessTime", "/testRoot");
+    checkOutput(
+        "             10   NOT_PERSISTED .+ .+ 100% " + oldFileName,
+        "             10   NOT_PERSISTED .+ .+ 100% " + oldFileRecentlyAccessed);
+  }
+
+  /**
+   * Tests ls command with sort by access time option.
```

Comment:
incorrect javadoc. Actually typically we don't javadoc test function any more, unless the test is very tricky

Reply:
and can you add test for `--timestamp`?

Reply:
Done